pkg/proxy/ipvs/meta_proxier.go

```diff
@@ -30,6 +32,8 @@ import (
 type metaProxier struct {
 	ipv4Proxier proxy.Provider
 	ipv6Proxier proxy.Provider
+	// TODO(imroc): implement node handler for meta proxier.
```

Comment:
FYI: https://github.com/kubernetes/kubernetes/pull/84089

Reply:
Thanks for the information, will add support to mata proxier support after endpointslice is supported in meta proxier.